registry/deploying.md

```diff
@@ -414,7 +414,7 @@ secrets.
       --entrypoint htpasswd \
       httpd:2 -Bbn testuser testpassword > auth/htpasswd
     ```
-    
```

Comment:
The `registry/` directory moved back to upstream https://github.com/distribution/distribution/pull/3397

(we still need to remove it here and/or set up a job to sync)

In the meantime, we should disable linting for this directory